test/set/built_set_test.dart

```diff
@@ -19,18 +19,6 @@ void main() {
       expect(set.isNotEmpty, isFalse);
     });
 
-    test('throws on attempt to create BuiltSet<dynamic>', () {
```

Comment:
maybe just replace expectations for these test cases instead of removing?

Reply:
Eh--seems a bit of a weird thing to test--it's true it's a functionality change, but it's hard to _accidentally_ disallow `dynamic`, since it's just an explicit check :)

I dunno--up to you :)

Reply:
SG :)